internal/validator/default/default_validator_test.go

```diff
@@ -333,3 +336,140 @@ func Test_needExtention(t *testing.T) {
 		})
 	}
 }
+
+func Test_cumulativeCheckFees(t *testing.T) {
```

Comment:
Also here a table based test would make it smaller and easier to spot the differences between each test case

Reply:
I changed them, but I'm not sure if they are more readable than before. Let me know what you think about the new form